src/Modules/OrchardCore.Commerce/Liquid/AddressFieldEditorViewModelConverterFilter.cs

```diff
@@ -0,0 +1,31 @@
+using Fluid;
+using Fluid.Values;
+using OrchardCore.Commerce.Fields;
+using OrchardCore.Commerce.ViewModels;
+using OrchardCore.Liquid;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Commerce.Liquid;
+
+public class AddressFieldEditorViewModelConverterFilter : ILiquidFilter
+{
+    public ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArguments arguments, LiquidTemplateContext context)
+    {
+        if (input?.ToObjectValue() is not CheckoutViewModel checkoutViewModel) return new ValueTask<FluidValue>(input);
+
+        var cityName = arguments["city_name"].Or(arguments.At(0)).ToStringValue();
+
+        if (arguments["address_field"].Or(arguments.At(0)).ToObjectValue() is not AddressField addressField ||
+            string.IsNullOrEmpty(cityName)) return new ValueTask<FluidValue>(input);
```

Comment:
Both are using argument index 0. If you pass in two positional arguments this will fail. If you pass city_name as named and the address field as positional, then it might work, but we shouldn't encourage mixing named and positional arguments.

Reply:
Ahh, this was just a copy paste mistake...